### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by alexst07**
Remove unused variables on mlir/lib/Dialect/Linalg/Transforms/LowerToLLVMDialect.cpp

### CODE REVIEW

File: lib/Dialect/Linalg/Transforms/LowerToLLVMDialect.cpp [hunk 1]
```diff
@@ -594,10 +594,7 @@ class TransposeOpConversion : public LLVMOpLowering {
     auto tranposeOp = cast<TransposeOp>(op);
     BaseViewConversionHelper helper(op, tranposeOp.getViewType(), rewriter,
                                     lowering);
-    IndexType indexType = helper.indexType;
-    ViewType viewType = helper.viewType;
-    LLVMType elementTy = helper.elementTy, int64Ty = helper.int64Ty,
-             viewDescriptorPtrTy = helper.viewDescriptorPtrTy;
```
Comment (joker-eph) [inline lib/Dialect/Linalg/Transforms/LowerToLLVMDialect.cpp]: Seems like some of these (like `indexType`) can be removed as a member of `BaseViewConversionHelper` as well?

Comment (alexst07) [reply lib/Dialect/Linalg/Transforms/LowerToLLVMDialect.cpp]: Yes, there are some attributes that no other part of the code is using. I will remove it and update the pull request.

Comment (joker-eph) [reply lib/Dialect/Linalg/Transforms/LowerToLLVMDialect.cpp]: Thanks

**Comment by nicolasvasilache** [comment]
Thanks for the cleanup, didn't get time to finish refactoring that piece
before leaving on PTO :)

**Pull request closed without merging**